core/views/browse.py

```diff
@@ -173,19 +173,14 @@ def issue_pages_rdf(request, lccn, date, edition):
 @cache_page(settings.DEFAULT_TTL_SECONDS)
 @vary_on_headers('Referer')
 def page(request, lccn, date, edition, sequence, words=None):
-    fragments = []
+    mutable_copy_of_get = request.GET.copy()
```

Comment:
This doesn't appear to be necessary any more: the use below can just be `request.GET.urlencode()`